libfaketime/__init__.py

```diff
@@ -103,17 +104,23 @@ def __init__(self, datetime_spec, only_main_thread=True):
             _datetime = dateutil.parser.parse(datetime_spec)
 
         self.time_to_freeze = _datetime  # freezegun compatibility
-        self.libfaketime_spec = _datetime.strftime('%Y-%m-%d %T %f')
 
     def _should_fake(self):
         return self.only_main_thread and threading.current_thread().name == 'MainThread'
 
-    def __enter__(self):
+    def _start_fake(self):
+        self.libfaketime_spec = self.time_to_freeze.strftime('%Y-%m-%d %T %f')
         if self._should_fake():
             begin_callback(self)
             self._prev_spec = os.environ.get('FAKETIME')
             os.environ['FAKETIME'] = self.libfaketime_spec
 
+    def tick(self, delta=datetime.timedelta(seconds=1)):
+        self.time_to_freeze += delta
+        self._start_fake()
```

Comment:
Is it intentional for tick() to start faking too? The examples in freezegun's docs seem to suggest it should only be called after start/enter have been used.